### ISSUE TITLE
Unimplemented syscalls

### ISSUE THREAD

**Issue by gandalfthewhite19890404** [obhq/obliteration#245]
Unimplemented syscalls:
- [ ] syscall 598
`syscall 598 at 0x0000000000001d7a on /system/common/lib/libkernel.sprx', kernel\src\syscalls\mod.rs:41:18`

**Comment by ultimaweapon** [comment]
Better to create one issue for each of it so it can be closed automatically when the implemented PR is merged.

**Comment by gandalfthewhite19890404** [comment]
> Better to create one issue for each of it so it can be closed automatically when the implemented PR is merged.

Autoclosing? How?

**Comment by ultimaweapon** [comment]
Here is how it is working: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Comment by ultimaweapon** [comment]
This one is quite easy to implement. The problem is there is a crash after this so I need to fix this crash before I can push the implementation for this.

**Comment by ultimaweapon** [comment]
It is crashing at `__orbis_rtld_wlock_acquire` in `libkernel` because the relocation is not applied yet.

**Comment by VocalFan** [comment]
Yep, happens on Amplitude, Project Diva X, and Dancing In All Night, just a universal thing.

**Comment by VocalFan** [comment]
Also sup, Ultima! Got anything you want me to do? :P

**Comment by ultimaweapon** [comment]
No idea yet.

**Comment by VocalFan** [comment]
Just saying, I think Slint would be a lot nicer than Qt... Finally go full Rust, really.
